0-common/scala-core/iglu-core-circe/src/main/scala/com.snowplowanalytics.iglu.core.circe/CirceIgluCodecs.scala

```diff
@@ -14,8 +14,9 @@ package com.snowplowanalytics.iglu.core.circe
 
 // Cats
 import cats.data._
-import cats.std.option._
+import cats.instances.option._
```

Comment:
Feel free to replace all three bottom imports with `cats.implicits._`. We're now trying to mimic cats' approach.